samples/Azure.Management.Storage/Generated/BlobContainer.cs

```diff
@@ -426,8 +426,10 @@ public async virtual Task<BlobContainerObjectLevelWormOperation> ObjectLevelWorm
             scope.Start();
             try
             {
+                var additionalOptions = new ChangeTrackingDictionary<string, string>();
```

Comment:
Lets make this a readonly class field of type string vs Dictionary and read it from the rest operations since the value is already evaluated there.

As for how to get the value into NextLinkOperationImplementation I prefer if we didn't add a new ctor param everywhere let me explore some ideas.